code/modules/vending/_vending.dm

```diff
@@ -185,6 +185,7 @@
 	/// used for narcing on underages
 	var/obj/item/radio/sec_radio
 
+	voice_filter = "aderivative"
```

Comment:
you should really put this before all of the new variable declarations on this type